docs/navigation/pages/navigation.md

```
@@ -30,106 +30,154 @@ There are `PagesNavigator` [extension functions](https://github.com/arkivanov/De
+    navigation.clear()
+    ```
+    
+    ```title="After"
+    []
+    ```
+
+### setItems
+
+!!! note
+
+    Available since version `3.4.0-alpha02`.
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)